aten/src/ATen/native/native_functions.yaml

```diff
@@ -3394,6 +3394,11 @@
     CPU: channel_shuffle
     QuantizedCPU: channel_shuffle_quantized_cpu
 
+- func: native_channel_shuffle(Tensor self, int groups) -> Tensor
```

Comment:
Hey @VitalyFedyunin, it looks like this op was added without any python docs. For the upcoming release, can either mark the op as private or add docs for it, and cherry-pick into the release branch?